### DISCUSSION TITLE
Diagram from markdown

### DISCUSSION THREAD

**Discussion by GiusFr** [jgraph/drawio#4173] [General]
Hi
I need to generate dinamically web pages from my application, in order to show diagrams.
My application is able to generate markdown code and, until now, I obtain the web page and the diagram using mermaid library
For example: my web page contains the following source:

<html>
<head>
<title>Flow chart </title>
<script src=""https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js""></script>
<meta charset=""UTF-8"">
</head>
<body>
<div   class='mermaid'>
graph TD
A -->B
</div>
</body>
</html>

The question is: it'is possible to obtain the same thing using appdiagram libraries and a specific markdown syntax, without the need to connect to the site https://apps.diagram.net and make the diagram from scratch?
Thanks

**Comment by davidjgraph** [comment]
This sounds like a question better directed at mermaid.js if it doesn't involve us.

**Comment by GiusFr** [comment]
My intention was to use appdiagram as an alternative to Mermaid. So I had asked if it was possible to dynamically generate web pages containing markdown code in order to obtain appdiagram diagrams

**Reply by davidjgraph** [reply]
That's not a description that sounds like draw.io functionality, there's no markdown support.
